Master qa 31886

ksuetani opened the pull request.

david-truong committed:
LPS-73056  new create token command and download bundle via token in initBundle

ksuetani committed:
LRQA-31886 Add macro to navigate to the login page

ksuetani committed:
LRQA-31886 New test for 3 bundle SAML single log out

liferay-continuous-integration commented:
Pull request test invoked at http://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest(master).

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest(master)/1174/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-7.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1174/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h1>All tests PASSED.</h1><p>Build Time: 40 minutes 26 seconds 109 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/51dbc5695ac5d6348d176c3986c7ca547862d298">51dbc5695ac5d6348d176c3986c7ca547862d298</a></p><h4>Job Summary:</h4><ul><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest(master)/1174/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-dist(master)/522/">test-portal-acceptance-pullrequest-dist(master)/jdk8</a></li></ul><h5>For more details click <a href="https://test-1-7.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1174/jenkins-report.html">here</a>.</h5></html><h5>Test bundle downloads:</h5><ul><li><a href="https://test-1-7.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1174/bundles/liferay-portal-bundle-tomcat.zip">liferay-portal-bundle-tomcat.zip</a></li></ul>

ksuetani committed:
LRQA-31886 Update outdated SAML paths

anthony-chu committed:
LRQA-32829 Move testPortalInstance validation into tear down macro

linolaoj committed:
LPS-72729 update asset without validation for default frontpage

ctampoya committed:
LRQA-32827 Move Forms <if> conditonals into macros

ctampoya committed:
LRQA-32827 Move DDL <if> condtionals into macros

ctampoya committed:
LRQA-32827 Move Calendar <if> conditionals into macros

robertoDiaz committed:
LPS-72729 Do not add asset entry service logic in wiki page service

patriciajeanperez committed:
LRQA-32898 Move conditional to a macro

ksuetani committed:
LRQA-32891 Remove conditionals from administration testcases

anthony-chu committed:
LRQA-32829 Only tear down portal instances

alee8888 committed:
LRQA-32898 SF

alee8888 committed:
LRQA-32891 SF

Commit:
LPS-72918 blind copy tasks-portlet from liferay-plugins-ee (6.2.x)

Commit:
LPS-72918 adjust build.xml

Commit:
LPS-72918 change imports portal.model to portal.kernel.model

Commit:
LPS-72918 change imports portal.security to portal.kernel.security

Commit:
LPS-72918 change imports portal.theme to portal.kernel.theme

Commit:
LPS-72918 change imports portal.util to portal.kernel.util

Commit:
LPS-72918 change imports for PortletURLFactoryUtil to portal.kernel.portlet.PortletURLFactoryUtil

Commit:
LPS-72918 change imports portal.service to portal.kernel.service

Commit:
LPS-72918 change imports counter.service to counter.kernel.service

Commit:
LPS-72918 remove compat from imports

Commit:
LPS-72918 add references to service.xml

Commit:
LPS-72918 rebuild service

Commit:
LPS-72918 adjust TasksEntryLSI to use references

Commit:
LPS-72918 change imports portlet.asset to asset.kernel

Commit:
LPS-72918 change imports portlet.messageboards to message.boards.kernel

ksuetani committed:
LRQA-31886 Split SAML macro into separate IdP and SP configuration macros

Commit:
LPS-72918 change imports portlet.social to social.kernel

Commit:
LPS-72918 adjust imports for exceptions

anthony-chu committed:
LRQA-33007 Pass in index instead of list of users

anthony-chu committed:
LRQA-33007 Update user email addresses

anthony-chu committed:
LRQA-33007 SF

anthony-chu committed:
LRQA-33007 Define user identification from email address

robertoDiaz committed:
LPS-72729 Create an exclusion rule for this case

Commit:
LPS-72918 adjust gitignore to include service.jar

Commit:
LPS-72918 handle leftover compile issues

Commit:
LPS-72918 remove shard* as it doesn't exist in 7.0.x

Commit:
LPS-72918 fix portal compatibility version

Commit:
LPS-72918 adjust imports for jsps

Commit:
LPS-72918 adjust method calls for newer api

Commit:
LPS-72918 sort imports on init.jsp

Sharryxue committed:
LRQA-33018 Increase the while loop

linolaoj committed:
LPS-72729 adding integration test

robertoDiaz committed:
LPS-72729 Add more tests

Commit:
LPS-72918 setup-eclipse

Commit:
LPS-72918 add gitignore

robertoDiaz committed:
LPS-72729 Since AssetEntryException is not serializable, it could not be used with the @Test(expected) annotation, we could use this hack to assert that this exception is been thrown.

robertoDiaz committed:
LPS-72729 SF

anthony-chu committed:
LRQA-33005 Use generic pathlib locator

stmerriss committed:
LRQA-33180 Add functionality to compare numbers to poshi runner

Preston-Crary committed:
LPS-73073 Reduce init.jsp overhead of IconTag.

Preston-Crary committed:
LPS-73103 Remove performance penalty of SocialBookmarksTag when setting social.bookmark.types property to nothing.

mbowerman committed:
LPS-73061 Add the pound character to the stop chars, since we don't want to parse the anchor as part of the URL

david-truong committed:
LPS-73056 update mojos to support new commands

david-truong committed:
LPS-73056 update ant tasks for new commands

Preston-Crary committed:
LPS-73103 Remove build trigger for social bookmarks hook, this should not be installed by default.

Preston-Crary committed:
LPS-73068 This method is already read only since we deligate through the advice for the real buffered write operation.

Preston-Crary committed:
LPS-73068 Build Service.

Ithildir committed:
LPS-73056 Wordsmith

Ithildir committed:
LPS-73056 Use a string as type for password

Ithildir committed:
LPS-73056 SF

Ithildir committed:
LPS-73056 The HTTP client is already authenticated

Ithildir committed:
LPS-73056 Allow to customize the token file location

Ithildir committed:
LPS-73056 No need to have it protected

Ithildir committed:
LPS-73056 Simplify

Ithildir committed:
LPS-73056 Have a constant with default value only if reused

Ithildir committed:
LPS-73056 Allow to customize the token URL, only for testing

Ithildir committed:
LPS-73056 Create util method

Ithildir committed:
LPS-73056 Test command

Preston-Crary committed:
LPS-73072 Reduce init.jsp overhead of UserDisplayTag.

Ithildir committed:
LPS-73056 Add missing Ant task metadata

Ithildir committed:
LPS-73056 More flexible token JSON parsing

Ithildir committed:
LPS-73056 Test Ant task

Ithildir committed:
LPS-73056 Test Mojo

Ithildir committed:
LPS-73056 Semver portal-tools-bundle-support

Commit:
LPS-72918 manual sf

Preston-Crary committed:
LPS-73074 Reduce init.jsp overhead of IconMenuTag.

Preston-Crary committed:
LPS-73095 Avoid using PropertyDescriptors in freemarker themes. These are the only two cases that cause freemarker to look up the method this way, we should always just call the method directly.

Preston-Crary committed:
LPS-73095 layout var cannot ever be null since we reference it unconditionally in this file.

Preston-Crary committed:
LPS-73095 Auto SF.

Ithildir committed:
LPS-73056 Consolidate all default values in a constants class

Ithildir committed:
LPS-73056 Allow to set token creation parameters via Maven

Ithildir committed:
LPS-73056 Allow to set the token file when downloading from Maven

Ithildir committed:
LPS-73056 Fix wrong error message

Ithildir committed:
LPS-73056 The util method already checks if the dir exists or not

Ithildir committed:
LPS-73056 Remove util method and simplify

Ithildir committed:
LPS-73056 Remove download command, we don't need it for now

Ithildir committed:
LPS-73056 Remove unnecessary abstract mojo

Ithildir committed:
LPS-73056 Goal names have dashes

Ithildir committed:
LPS-73056 Regen Maven plugin descriptor

tinatian committed:
LPS-72505 Deprecated unused methods

brianchandotcom committed:
LPS-72918 match Testray

Ithildir committed:
LPS-73056 Move file constant into the constants class

Ithildir committed:
LPS-73056 Export constants class

brianchandotcom committed:
LPS-72918 lang

brianchandotcom committed:
LPS-72918 not needed

brianchandotcom committed:
LPS-72918 manual SF

Ithildir committed:
LPS-73056 Move default values to constants class

Commit:
LPS-72918 change CustomSQLUtil per LPS-64238

Ithildir committed:
LPS-73506 Remove outdated sample

tinatian committed:
LPS-72505 Add PortletIdCodecTest

Commit:
LPS-72918 selectively apply changes from liferay-plugins-ee master

Commit:
LPS-72918 auto sf

Commit:
LPS-72918 change import for MVCPortlet

jkappler committed:
LPS-72709 build.gradle; change source and target compatibility for the module

jkappler committed:
LPS-72709 Take into account the default decorator from the theme when there's none defined

Commit:
LPS-72918 copy/edit files from other legacy portlets in modules dir

Commit:
LPS-72918 add CustomSQL jar

mbowerman committed:
LPS-73061 Apply the same solution to BaseTextExportImportContentProcessor

Even though this class is deprecated, several of our components still use it. So we have to have duplicated code in DefaultTextExportImportContentProcessor and BaseTextExportImportContentProcessor in the meantime.

pat270 committed:
LPS-72597 Update Lexicon to 1.0.20

Commit:
LPS-72918 squelch needless NoSuchUser exception

shuyangzhou committed:
LPS-73103 Simply use _SOCIAL_BOOKMARK_TYPES as default value for _types

tinatian committed:
LPS-72505 Stop using PortletConstants, use PortletIDCodec instead

shuyangzhou committed:
LPS-73103 The jsp is expecting String[], just feed it with that.

mbowerman committed:
LPS-73061 Add test cases

lesliewong92 committed:
LRQA-33181 Reword to make parallel

ctampoya committed:
LRQA-33265 Choose a different Doc Type

Ithildir committed:
LPS-72705 Update changelog

michaelhashimoto committed:
LRQA-32571 Overwrite the current JMX Remote Port for additional bundles

Preston-Crary committed:
LPS-73123 Avoid fetching and updating AssetEntry's AssetTags when there are no tags being changed.

shinnlok committed:
LPS-73124 Task fails after with newly added table

shinnlok committed:
LPS-73124 Add test

brianchandotcom committed:
users-admin-demo-data-creator-api 1.3.0 apply

brianchandotcom committed:
Apply portal-kernel 2.33.0

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
roles-admin-demo-data-creator-api 1.0.1 apply

brianchandotcom committed:
artifact:ignore wiki-engine-mediawiki 2.0.10 prep next

brianchandotcom committed:
artifact:ignore document-library-service 1.1.23 prep next

brianchandotcom committed:
artifact:ignore document-library-web 1.4.0 prep next

brianchandotcom committed:
artifact:ignore portal-configuration-module-configuration 2.0.9 prep next

brianchandotcom committed:
artifact:ignore portal-configuration-settings 2.0.11 prep next

brianchandotcom committed:
artifact:ignore portal-search-elasticsearch 2.1.6 prep next

brianchandotcom committed:
artifact:ignore roles-admin-demo-data-creator-api 1.0.1 prep next

brianchandotcom committed:
artifact:ignore users-admin-demo-data-creator-impl 1.0.9 prep next

brianchandotcom committed:
artifact:ignore product-navigation-product-menu-web 2.0.15 prep next

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
packageinfo 2.34.0

brianchandotcom committed:
artifact:ignore roles-admin-demo-data-creator-impl 1.0.0 prep next

brianchandotcom committed:
artifact:ignore roles-admin-demo-data-creator-impl 1.0.0 prep next

brianchandotcom committed:
artifact:ignore users-admin-demo 1.0.2 prep next

brianchandotcom committed:
dynamic-data-mapping-web packageinfo

brianchandotcom committed:
portal-workflow-kaleo-definition-impl packageinfo

brianchandotcom committed:
portal-workflow-task-web packageinfo

brianchandotcom committed:
journal-content-web packageinfo

brianchandotcom committed:
Sync

brianchandotcom committed:
Sync

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/document-library/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/message-boards/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/microblogs/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/ratings/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/social/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/calendar/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/contacts/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-js/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-theme/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/login/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/mobile-device-rules/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/petra/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-cache/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-configuration/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-search/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/roles/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/server/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/users-admin/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/marketplace/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/static/portal-osgi-web/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/asset/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/export-import/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/layout/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/nested-portlets/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/portlet-display-template/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/product-navigation/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/site/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/staging/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/trash/.gitrepo'.

hhuijser committed:
LPS-72705 Change regex to catch more cases where array fits on single line

hhuijser committed:
LPS-72705 Auto SF

adolfopa committed:
LPS-72729 Avoid indirect dependency

adolfopa committed:
LPS-72729 Use a more specific exception

To both avoid the Arquillian serialization problem and the possibility
of unintendedly masking other errors, use a taylor made exception for
this test case.

ealonso committed:
LPS-73132 Reorder table columns

ealonso committed:
LPS-72709 SF

ambrinchaudhary committed:
LPS-69296 This JSP must contain all the nav taglib structure (nav-bar, nav, nav-item, nav-bar-search).
Otherwise with the <liferay-util:include> does not work properly.

ambrinchaudhary committed:
LPS-69296 Adapt views

Commit:
LPS-73061 Make it more obvious

kveress committed:
LRQA-33289 Update enablePageInSitePagesVariation

Commit:
LPS-72780 Prevent undefined value from being pushed to array

brianchandotcom committed:
LPS-72709 Better name. I wouldn't call it portletDecoratorsList because we have another var called portletDecorators that is already a List. This is really a filtered list.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/portlet-configuration/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/message-boards/.gitrepo'.

Ithildir committed:
LPS-72875 Semver asset-publisher-web

brianchandotcom committed:
LPS-72505 I prefer "

brianchandotcom committed:
Revert "LPS-73103 Remove build trigger for social bookmarks hook, this should not be installed by default."

This reverts commit 89fb9e70bf26597ccb6091d8f11b46baab252a91.

Ithildir committed:
LPS-73124 Regen Maven plugin descriptor

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/asset/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/wiki/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-theme/.gitrepo'.

Ithildir committed:
LPS-72365 Avoid deleting the "osgi/test" directory with the "clean" task

Preston-Crary committed:
LPS-73072 Add back missing contentType.

Preston-Crary committed:
LPS-73074 Add back missing contentType.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/dynamic-data-lists/.gitrepo'.

Ithildir committed:
LPS-73139 Use latest version of JRuby

Ithildir committed:
LPS-73139 cache

Ithildir committed:
LPS-73139 Invoke JRuby via proxy

shuyangzhou committed:
LPS-72505 SF

brianchandotcom committed:
LPS-72918 No need to change the build number

brianchiu committed:
LRQA-33306 This would never run anyways

brianchiu committed:
LRQA-33306 Extract out the exception to the norm

brianchiu committed:
LRQA-32827 One more setup

Ithildir committed:
Revert "LPS-71789 Ignore failing tests for now"

This reverts commit f456a5d2057237664cb762d42e36551f3b7c9894.

Ithildir committed:
LPS-73056 Avoid going through Squid during testing

Ithildir committed:
LPS-73147 This is already the default value

See https://github.com/bndtools/bnd/commit/b871334e064aa46528bea15419969c31c5fa9ac5

Ithildir committed:
LPS-73147 Update changelog

Ithildir committed:
LPS-73148 Show warning message if no Table classes are found

michaelhashimoto committed:
LRQA-33312 Do not force update upstream to HEAD for upstream branches

michaelhashimoto committed:
LRQA-33312 Ignore 'FAST_FORWARD' exceptions since there is no functional effect

brianchiu committed:
LRQA-32827 Move conditional into macro

michaelhashimoto committed:
LRQA-33312 SF

brianchandotcom committed:
LPS-73124 prep next, use latest

brianchandotcom committed:
LPS-73148 prep next, use latest

brianchandotcom committed:
LPS-72705 Prep next, use latest

brianchandotcom committed:
LPS-72705 Prep next, use latest

brianchandotcom committed:
Auto SF

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/blogs/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/bookmarks/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/document-library/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/message-boards/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/wiki/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/calendar/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/dynamic-data-lists/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-scheduler/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-security/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-security-sso/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-template/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/marketplace/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/static/portal-osgi-web/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/asset/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/site/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/staging/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/asset/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-js/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-theme/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/export-import/.gitrepo'.

Commit:
LPS-71147 Fix css of form settings modal

ksuetani committed:
LRQA-31886 Update 3 bundle SAML test to pass as expected

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/dynamic-data-lists/.gitrepo'.
